pepper-apis/src/main/java/org/broadinstitute/ddp/content/I18nContentRenderer.java

```diff
@@ -35,14 +27,10 @@
     public static final String DEFAULT_LANGUAGE_CODE = "en";
 
     private static final Logger LOG = LoggerFactory.getLogger(I18nContentRenderer.class);
-    private static final String TEMPLATE_NAME = "testTemplate";
-    private static final String STRING_RESOURCE_LOADER_CLASS_PROP = "string.resource.loader.class";
-    private static final String STRING_RESOURCE_LOADER_REPOSITORY_STATIC_PROP
-            = "string.resource.loader.repository.static";
+    private static final String TEMPLATE_NAME = "template";
```

Comment:
This ultimately ends up being a label in a log statement somewhere, so it doesn't really matter what this is.